Cargo.toml

```diff
@@ -20,6 +23,7 @@ rayon = "1.5.0"
 rustc-hash = "1.1.0"
 smallvec = "1.4.2"
 twox-hash = "1.6.0"
+simdjson-rust = {git = "https://github.com/SunDoge/simdjson-rust"}
```

Comment:
They wrote on github:
`This crate is currently updating to support simdjson 0.3.1! You can have a try and give feedback.`

I was wondering if we can use: https://crates.io/crates/simd-json

@kodieg @michaelmior

Reply:
I had issues getting simd-json to compile on my Mac which is why I stuck with this for now.